app/navigation/router.js

```diff
@@ -102,9 +104,14 @@ class Router extends React.Component {
                 };
             }
 
-            // NavigationExperimental only creates the correct panHandlers if the panHandlers prop === undefined
-            const panHandlers = navigationProps.allowSceneSwipe ? undefined : null; // eslint-disable-line
-
+            let panHandlers = null;
+            if (navigationProps.allowSceneSwipe) {
+                panHandlers = navigationPanResponder.forHorizontal({
+                    ...cardProps,
+                    gestureResponseDistance: 60, // sets the distance from the edge for swiping
```

Comment:
@jarredwitt I think this is still to close to the edge, and then we may want to do the same thing for the drawers?